openwisp_controller/config/base/vpn.py

```diff
@@ -205,13 +256,37 @@ def save(self, *args, **kwargs):
             self._generate_wireguard_keys()
         if self.subnet and not self.ip:
             self.ip = self._auto_create_ip()
-        super().save(*args, **kwargs)
+        if self._is_backend_type('zerotier'):
+            config = deepcopy(self.config['zerotier'][0])
+            config['name'] = self.name
+            if created:
+                self._create_zerotier_server(config)
+        try:
+            super().save(*args, **kwargs)
+        except Exception as exc:
```

Comment:
Small request for change for reasons of consistency with the rest of the codebase: 
```suggestion
        except Exception as e:
```